.gitlab-ci.yml

```diff
@@ -14,20 +14,16 @@ default:
       - node_modules/
     policy: pull-push
 
-
-
 build:
   stage: build
   image: node:22.11.0-alpine3.20
   script:
-      - yarn install --dev --frozen-lockfile
-      - yarn build
```

Comment:
I need that step. Not building the library in the pipeline was one of the reasons why the dist folder was missing in 0.1.4

Reply:
I think it is a result of resolving conflicts. Sorry. Actually I did not touch this file. There was no reason to do this)